cypress/integration/app/activateuser.spec.ts

```diff
@@ -0,0 +1,104 @@
+describe("Invite user", () => {
+  it("Enable SMTP setting", () => {
+    cy.setup();
+    cy.login();
+
+    cy.visit("/settings/organization");
+
+    cy.findByLabelText(/enable smtp/i).check({ force: true });
+
+    cy.findByLabelText(/sender address/i)
+      .click()
+      .type("rachel@fleetdm.com");
+
+    cy.findByLabelText(/smtp server/i)
+      .click()
+      .type("localhost");
+
+    cy.get("#port").click().type("{selectall}{backspace}1025");
+
+    cy.findByLabelText(/use ssl\/tls/i).check({ force: true });
+
+    cy.findByLabelText(/smtp username/i)
+      .click()
+      .type("rachelsusername");
+
+    cy.findByLabelText(/smtp password/i)
+      .click()
+      .type("rachelspassword");
+
+    cy.findByRole("button", { name: /update settings/i }).click();
+  });
+
+  it("Invite user", () => {
+    cy.findByRole("tab", { name: /^users$/i }).click();
+
+    cy.contains("button", /invite user/i).click();
+
+    cy.findByLabelText(/name/i).click().type("Ash Ketchum");
+
+    cy.findByLabelText(/email/i).click().type("ash@fleetdm.com");
+
+    cy.findByRole("button", { name: /invite/i }).click();
+
+    cy.wait(3000);
+  });
+
+  let inviteLink;
+
+  it("Accept invite and create user", () => {
+    const regex = /\/login\/invites\/[a-zA-Z0-9=?%&@._-]*/gm;
+
+    cy.getEmails().then((response) => {
+      expect(response.body.items[0].To[0]).to.have.property("Domain");
+      expect(response.body.items[0].To[0].Mailbox).to.equal("ash");
+      expect(response.body.items[0].To[0].Domain).to.equal("fleetdm.com");
+      expect(response.body.items[0].From.Mailbox).to.equal("rachel");
+      expect(response.body.items[0].From.Domain).to.equal("fleetdm.com");
+      inviteLink = response.body.items[0].Content.Body.match(regex);
```

Comment:
If you add an `expect` here for the expected length of the match array, someone running the test will get a more helpful error message. Something like 

```suggestion
      const match = response.body.items[0].Content.Body.match(regex);
      expect(match).to.have.length(1);
      inviteLink = match[0];
```

I think it also helps the name of the variable `inviteLink` match the structure of what's being stored in it. Then you'll also need to change line 63.